docs/content/references/cli/client.mdx

```diff
@@ -188,6 +188,211 @@ $ sui client dynamic-field 0x5
 ╰─────────────┴───────────────────────────────────────────────────────────────────────────────────────────╯
 ```
 
+### Send SUI or objects
+
+In this example, let's see how to transfer SUI or transfer an object from one address to another. First all, there two main commands for sending SUI or transfering objects: Pay and Transfer. 
```

Comment:
```suggestion
In this example, let's see how to transfer SUI or transfer an object from one address to another. First of all, there are two main commands for sending SUI or transferring objects: Pay and Transfer. 
```

If "Pay" and "Transfer" are referring to commands, let's use `pay` and `transfer`.